opp_path_selection/src/path_selection_artist.cpp

```diff
@@ -0,0 +1,487 @@
+/*
+ * Copyright 2018 Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "opp_path_selection/path_selection_artist.h"
+
+#include <pcl/conversions.h>
+#include <pcl/point_cloud.h>
+#include <pcl/PolygonMesh.h>
+#include <pcl/surface/simplification_remove_unused_vertices.h>
+
+#include <eigen_conversions/eigen_msg.h>
+#include <geometry_msgs/PointStamped.h>
+#include <opp_msgs_serialization/serialize.h>
+#include <ros/package.h>
+#include <shape_msgs/Mesh.h>
+#include <tf/tf.h>
+#include <tf_conversions/tf_eigen.h>
+
+#include "opp_path_selection/path_selector.h"
+#include "opp_path_selection/path_selector_parameters.h"
+#include <opp_msgs/GetPathSelectionMesh.h>
+
+namespace opp_path_selection
+{
+const static double TF_LOOKUP_TIMEOUT = 5.0;
+const static std::string MARKER_ARRAY_TOPIC = "path_markers";
+const static std::string CLICKED_POINT_TOPIC = "clicked_point";
+const static std::string CLEAR_PATH_POINTS_SERVICE = "clear_path_selection_points";
+const static std::string COLLECT_PATH_POINTS_SERVICE = "collect_path_selection_points";
+const static std::string path_selection_config_file = ros::package::getPath("opp_path_selection") + "/config/"
```

Comment:
This seems like something that should be passed in to a function or constructor as a string argument rather than burying a hard-coded file path in a source file